content/chat.adoc

```diff
@@ -3,6 +3,30 @@ layout: simplepage
 title: Chat
 ---
 
+= Gitter
```

Comment:
This does not actually work.

    asciidoctor: ERROR: chat.adoc: line 28: level 0 sections can only be used when doctype is book

The page looks really weird right now:

> ![screen shot](https://user-images.githubusercontent.com/1831569/53048085-3aee2300-3494-11e9-8b24-57ad101f5e76.png)

I recommend you build the site during development, or check the resulting output after it's merged, in the future.

Reply:
Fixed in https://github.com/jenkins-infra/jenkins.io/pull/2119